.github/workflows/_publish-container.yml

```diff
@@ -46,18 +43,5 @@ jobs:
       - name: Login to ACR
         run: az acr login --name ${{ vars.CONTAINER_REGISTRY_NAME }}
 
-      - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
       - name: Build and push container image
-        working-directory: ${{ inputs.docker_context }}
-        run: |
-          docker buildx create --use
-          docker buildx build \
-            --platform linux/amd64,linux/arm64 \
-            --build-arg VERSION=${{ inputs.version }} \
-            -t ${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:${{ inputs.version }} \
-            -t ${{ vars.CONTAINER_REGISTRY_NAME }}.azurecr.io/${{ inputs.image_name }}:latest \
-            -f ${{ inputs.docker_file }} \
-            --push .
-          docker buildx rm
+        run: dotnet publish ${{ inputs.api_location }} -t:PublishContainer -r linux-x64 -p ContainerImageTags='"${{ inputs.version }};latest"'
```

Comment:
I think you need to add this to actually publish it to ACR.

`-p ContainerRegistry=${{ vars.CONTAINER_REGISTRY_NAME }}